crates/opencascade/src/primitives/shape.rs

```diff
@@ -631,6 +631,14 @@ impl Shape {
         self.inner.pin_mut().set_global_translation(&location, false);
     }
 
+    /// Copy `self` using the upper 3x4 matrix of `transform`.
+    pub fn transform(&self, transform: &DMat4) -> Self {
+        let t = crate::transform::gp_trsf(transform);
+        // NOTE: Setting the copy argument does not seem to have an effect.
+        let mut res = ffi::BRepBuilderAPI_Transform_ctor(&self.inner, &t, true);
```

Comment:
Could you pull `true` into a named local variable?
```suggestion
        let copy = true;
        let mut res = ffi::BRepBuilderAPI_Transform_ctor(&self.inner, &t, copy);
```

Reply:
Will do! Sorry for taking so long. Very busy at work the past several weeks!